release.sh

```diff
@@ -90,11 +90,10 @@ export GOFLAGS="-ldflags=-s -ldflags=-w"
 export GITHUB_TOKEN=""
 readonly IMAGES_REFS_FILE="${IMAGES_REFS_FILE:-$(mktemp -d)/images_refs.txt}"
 
-# Convenience function to run the hub tool.
-# Parameters: $1..$n - arguments to hub.
-function hub_tool() {
-  # Pinned to SHA because of https://github.com/github/hub/issues/2517
-  go_run github.com/github/hub/v2@363513a "$@"
+# Convenience function to run the GitHub CLI tool `gh`.
+# Parameters: $1..$n - arguments to gh.
+function gh_tool() {
+  go install github.com/cli/cli/v2/cmd/gh@v2.65.0
```

Comment:
```suggestion
  go_run github.com/cli/cli/v2/cmd/gh@v2.65.0 "$@"
```

Go install, just installs. It can't be used to run the tool.